docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -0,0 +1,266 @@
+
+Nowadays, we can use the following HTML code:
+
+```
+<video width="500" controls>
+	<source src="https://www.mapbox.com/bites/00188/patricia_nasa.webm" type="video/webm">
+	<source src="https://www.mapbox.com/bites/00188/patricia_nasa.mp4" type="video/mp4">
+</video>
+```
+
+To display this video:
+
+<video width="500" controls>
+<source src="https://www.mapbox.com/bites/00188/patricia_nasa.webm" type="video/webm">
+<source src="https://www.mapbox.com/bites/00188/patricia_nasa.mp4" type="video/mp4">
+</video>
+
+If a video can be shown in a webpage in this way, then Leaflet can display it inside a map. It is important that the videos are prepared in such a way that they will fit the map: The video should have a "north-up" orientation, and its proportions should fit the map. If not, it will look out of place.
+
+#### Creating a map
+
+First of all, create a Leaflet map and add a background `L.TileLayer` in the usual way:
+
+```
+var map = L.map('map').setView([37.8, -96], 4);
+
+L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=' + mapboxAccessToken, {
+	id: 'mapbox/satellite-v9',
+	attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, ' +
+		'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+	tileSize: 512,
+	zoomOffset: -1
+}).addTo(map);
+```
+
+Then, we'll define the geographical bounds that the video will cover. This is an instance of [`L.LatLngBounds`](/reference.html#latlngbounds), which is a rectangular shape:
+
+```
+var latLngBounds = L.latLngBounds([[32, -130], [13, -100]]);
+```
+If you want to see the area covered by a `LatLngBounds`, use a [`L.Rectangle`](/reference.html#rectangle):
+```
+L.rectangle(latLngBounds).addTo(map);
+
+map.fitBounds(latLngBounds);
+```
+
+{% include frame.html url="example-bounds.html" %}
+
+#### Adding the video overlay
+
+Adding a video overlay works very similar to adding an image overlay. 
+
+For a video overlay, just:
+
+* Use `L.videoOverlay` instead of `L.imageOverlay`
+* `L.videoOverlay` is used to load and display a video player over specific bounds of the map. Extends [`L.ImageOverlay`](/reference.html#imageoverlay). 
+A video overlay uses the [`<video>`](https://developer.mozilla.org/docs/Web/HTML/Element/video) HTML element.
+* Instead of the image URL, specify one video URL *or* an array of video URLs
+
+```
+var videoUrls = [
+	'https://www.mapbox.com/bites/00188/patricia_nasa.webm',
+	'https://www.mapbox.com/bites/00188/patricia_nasa.mp4'
+];
+var errorOverlayUrl = 'https://cdn-icons-png.flaticon.com/512/110/110686.png';
+var altText = 'Video of Hurricane Patricia from Satellite. Source: NASA'
+var latLngBounds = L.latLngBounds([[32, -130], [13, -100]]);
+
+var videoOverlay = L.videoOverlay(videoUrls, latLngBounds, {
+	opacity: 0.8,
+	errorOverlayUrl: errorOverlayUrl,
+	alt: altText,
```

Comment:
`alt` isn't supported on videos?

Reply:
> `alt` isn't supported on videos?

I thought it is supposed to work since this Option is inherited from ImageOverlay: https://leafletjs.com/SlavaUkraini/reference.html#videooverlay
but actually, it seems that it is not working

Reply:
remove altText for `videoOverlay`

Reply:
done